### ISSUE TITLE
Error return code when --verify fails

### ISSUE THREAD

**Issue by strawberryhacker** [microchip-pic-avr-tools/pymcuprog#38]
Request: return error code when --verify  fails

I am running this comand and captures the return code:

```
$ sudo pymcuprog write -d avr64ea48 -i updi --verify -f build/out.hex && echo Status code: $?
```

gives status code 0 when it fails

```
Writing flash...
Verifying flash...
pymcuprog.programmer - ERROR - Verify failed for flash memory:
pymcuprog.programmer - ERROR - Verify mismatch starting at location 0x000000: 0x0C vs 0xFF (is the memory section erased?)
Verification failed!
Done.
Status code: 0
```

and status code 0 when it succeeds

```
Writing flash...
Verifying flash...
OK
Done.
Status code: 0
```

**Comment by xedbg** [comment]
Oh dear, that is not good and should be fixed.  Logged as DSG-6409

**Comment by pic-avr-github-admin** [comment]
Fixed in 3.16 release.
